### ISSUE TITLE
Some results and questions

### ISSUE THREAD

**Issue by jianjun0407** [nihaomiao/MICCAI22_ADN#4]
Refer to some of your suggestions, have some results and questions, would like to discuss with you.
1、May I ask you which normalization method is used in data preprocessing? 
I tried two normalization methods, one was global maximum and minimum normalization, and the other was clip to [0,80] , then followed by maximum and minimum normalization. When  pretrain with your checkpoint, you can see a big difference in initial results. The result of clip first will be better. Did you clip during normalization? What is the clip range?
2、About the brain tissue segmentation network, why is the output channel 5 instead of 4? 
I think background, gray matter, white matter, cerebrospinal fluid, there are four categories.
![1](https://user-images.githubusercontent.com/19251556/226109471-900f7c7b-33f0-491e-b7ab-8f3f20302c06.png)
3、Do you still have the checkpoint of the brain tissue segmentation network(‘’gwm_seg_model‘’)?
4、The following results are my best results at present. There will be some anomalies in the test of some data. I am still investigating the reasons.
![2](https://user-images.githubusercontent.com/19251556/226109486-62aa3863-8d2e-4a57-88bc-deb78904d840.png)
![3](https://user-images.githubusercontent.com/19251556/226109491-253020c3-a668-4338-bb60-d1a8a4791e46.png)
![4](https://user-images.githubusercontent.com/19251556/226109498-c143fbf7-7e58-49cb-bed8-3f46d6b8f0f6.png)

**Comment by nihaomiao** [comment]
Hi, @jianjun0407,

Thanks a lot for your feedback!

1. The data preprocessing is performed by doctors instead of me so I am not sure about the concert details. But clipping the value is common because pixel value in CT has physical meaning. 

2. We apply off-the-shelf tools to the training dataset to generate background, GM, WM, and CSF, and we also utilize the stroke label to train an auxiliary segmentation model. So we have 5 channels here. 

3. Sure, I will check whether it is still on the server and send it to your email.

4. Could you also report the metrics we used in our paper? Some of the results are quite similar to our previous unused checkpoints. Maybe further training and tuning will have better performance. 

Bests
Haomiao

**Comment by jianjun0407** [comment]
Thank you for sharing!     
   As for retraining, I am now going to use other algorithms to obtain the label of brain tissue and train the model of brain tissue segmentation. This step is the premise of the overall network training, so I have not calculated the relevant metric in the paper.
    Yes, the above results were directly tested against your checkpoint by adjusting different data preprocessing methods. What I'm doing now is a step by step training process. 
    Looking forward to further communication with you.

**Comment by jianjun0407** [comment]
Thank you for sharing!     
   As for retraining, I am now going to use other algorithms to obtain the label of brain tissue and train the model of brain tissue segmentation. This step is the premise of the overall network training, so I have not calculated the relevant metric in the paper.
    Yes, the above results were directly tested against your checkpoint by adjusting different data preprocessing methods. What I'm doing now is a step by step training process. 
    Looking forward to further communication with you.
